identity/src/main/java/com/stripe/android/identity/IdentityVerificationSheet.kt

```diff
@@ -92,5 +96,31 @@ interface IdentityVerificationSheet {
             identityVerificationCallback: IdentityVerificationCallback
         ): IdentityVerificationSheet =
             StripeIdentityVerificationSheet(from, configuration, identityVerificationCallback)
+
+        /**
+         * Creates a [IdentityVerificationSheet] instance in a [Composable]. Which would be
+         * recreated if [configuration] or [identityVerificationCallback] changed.
```

Comment:
It shouldn't be recreated if `identityVerificationCallback` changes, but the latest callback is the one that's called (if you make the change suggested below).

Reply:
done